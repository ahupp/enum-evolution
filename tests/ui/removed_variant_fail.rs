use enum_evolution::enum_evolution;

enum_evolution! {
    enum Foo {
        Zero,
        One(usize),
    }

    derive Bar from Foo {
        remove Zero
    }
}

fn main() {
    let _ = Bar::Zero;
}
