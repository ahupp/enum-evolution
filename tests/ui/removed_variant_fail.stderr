error[E0599]: no variant or associated item named `Zero` found for enum `Bar` in the current scope
  --> tests/ui/removed_variant_fail.rs:15:18
   |
4  | /     enum Foo {
5  | |         Zero,
6  | |         One(usize),
7  | |     }
   | |_____- variant or associated item `Zero` not found for this enum
...
15 |       let _ = Bar::Zero;
   |                    ^^^^ variant or associated item not found in `Bar`
